Formula/kmod.rb

```diff
@@ -0,0 +1,36 @@
+class Kmod < Formula
+  desc "Linux kernel module handling"
+  homepage "https://git.kernel.org/?p=utils/kernel/kmod/kmod.git"
+  url "https://www.kernel.org/pub/linux/utils/kernel/kmod/kmod-29.tar.xz"
+  sha256 "0b80eea7aa184ac6fd20cafa2a1fdf290ffecc70869a797079e2cc5c6225a52a"
+  license all_of: ["LGPL-2.1-or-later", "GPL-2.0-or-later"]
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "gcc" => :build
+  depends_on "libtool" => :build
+  depends_on "m4" => :build
+  depends_on "make" => :build
```

Comment:
The build dependency list doesn't look right.
- I don't think `gcc` dependency is even being used since you haven't overridden the default compiler (GCC 5).
- `make` should already be available in build environment. Rare to need a dedicated dependency.
- Odd to need to run `libtoolize`. Have you tried just `configure` step? Usually packaged `configure` script shouldn't require `autoconf`, `automake`, `libtool`, or `m4`.

```suggestion
```

Reply:
Thanks, I was able to drop it, for some reason it did not pass CI initially that's why I went this route.